src/index.js

```diff
@@ -89,6 +89,7 @@ class JanusAdapter {
 
     this.publisher = null;
     this.occupants = {};
+    this.leftOccupants = new Set();
```

Comment:
I would have preferred to use `occupants` to track this, per a illegal-states-should-be-unrepresentable sort of heuristic, but I see why you did it this way and I don't really endorse refactoring a lot in here right now. (I expect this code to be totally torn up and thrown out when we implement Unified Plan support.)